components/jvm-build-service/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+resources:
+- https://github.com/redhat-appstudio/jvm-build-service/deploy/kcp?ref=bc1ecfaf82f1edaef5b4d80e53d093e371048fb7
+
+images:
+- name: hacbs-jvm-operator
+  newName: quay.io/redhat-appstudio/hacbs-jvm-controller
+  newTag: e927aa1b0dcd479ce0ad4f4a62a87b3b1b7dc423
```

Comment:
the operator will now be able to determine the image ref for the other jvm build service images built with this commit ^^ and use them